### ISSUE TITLE
[SF][1.25.1][Table][A11y][Expected Fixed Time: ASAP] Loss of focus and unable to access interactive content within each row in small screen size

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents#9288]
### Bug Description

https://github.com/SAP/ui5-webcomponents/issues/8385
This issue has not been handled. This issue is an reopened one of that.

### Expected Behaviour

There is no way to tab inside each row to access the links and quick card within each row in small screen size for example 320px

### Isolated Example

https://stackblitz.com/edit/github-xbtmwe?file=src%2FApp.tsx

### Steps to Reproduce

1. Click the URL
2. Change the screen size to 320px
3. When using tab key for access the accessible elements from each rows, you will find it cannot be access inside for each row

### Log Output, Stack Trace or Screenshots
<img width="259" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/156821927/5103a3d4-563a-41cf-b0c2-c92cfce00fda">

### Priority

None

### UI5 Web Components Version

1.22.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by kineticjs** [comment]
Hello, @SAP/UI5-webcomponents-topic-core, 

The reported issue is that the user cannot access the interactive controls inside the table rows with popin using keyboard..
The described issue also reproduces on the TableSelection.html test page:
-- On a wider screen (when **no** pop-in) the user can select the "Active" button using 'tab' key (given they previously focused the wrapping row)
-- On a narrower screen (when pop-in is rendered) the tab key does not focus the "Active" button

I could not find in the KBH spec an instruction to select interactive content inside table rows via tab key.
But I found the following recommendation for F2 key instead, which also seems unsupported in ui5-table in popin mode:

KBH spec excerpt:
"F2:
If focus is on a cell, and cell has focusable content, switch keyboard mode from “Navigation” mode to
“Edit” mode. Move focus to the first focusable content control."

Would you please comment if the observed behavior is a bug or a feature?

**Comment by wilson-xu-sap** [comment]
@ilhan007 @nnaydenow  Do you have a plan to merge this fix into version 1.xx?

**Comment by nnaydenow** [comment]
Hi @wilson-xu-sap,

The fix was downported with https://github.com/SAP/ui5-webcomponents/pull/9424 and it will be available with the next patch 1.24.8.

**Comment by wilson-xu-sap** [comment]
> Hi @wilson-xu-sap,
> 
> The fix was downported with #9424 and it will be available with the next patch 1.24.8.

Glad to hear that, thanks for your work. 😸

**Comment by wilson-xu-sap** [comment]
BTW, when will 1.24.8 release?

**Comment by ilhan007** [comment]
Hello @wilson-xu-sap  **every start of the month** we release:
- minor release for the main code line - 2.1, 2.2
- and patch for the maintenance code line - 1.24.8, 1.24.9

In case of urgency, we also provide patches sooner, on-demand.
That means either first days of Aug, or sooner in case of urgency.

Is the regular schedule fits your timelines? If not, please give us some details on the urgency matter.

BR,
ilhan

**Comment by wilson-xu-sap** [comment]
> Hello @wilson-xu-sap **every start of the month** we release:
> 
> * minor release for the main code line - 2.1, 2.2
> * and patch for the maintenance code line - 1.24.8, 1.24.9
> 
> In case of urgency, we also provide patches sooner, on-demand. That means either first days of Aug, or sooner in case of urgency.
> 
> Is the regular schedule fits your timelines? If not, please give us some details on the urgency matter.
> 
> BR, ilhan

Glad to hear that, thanks.

**Comment by wilson-xu-sap** [comment]
We will do the update work after your release.
